docs/reference/NuGet-Client-SDK.md

```diff
@@ -34,12 +34,33 @@ You can find the source code for these packages in the [NuGet/NuGet.Client](http
 
 ## Support policy
 
-All security bugs should be reported to the Microsoft Security Response Center (MSRC) at [https://aka.ms/opensource/security/create-report].
-Also see the [security policy in the NuGet.Client repo](https://github.com/NuGet/NuGet.Client/blob/dev/SECURITY.md).
+We are fully committed to supporting the most recent versions of the NuGet Client SDK packages.
```

Comment:
Given our `we don't guarantee API` stability disclaimer, I'm wondering if we should tone down the enthusiasm in the SDK section? :D 
wyt @donnie-msft?

Reply:
Agreed. Docs aren't the place to be excited. NOR ARE GITHUB REVIEWS! jk

Reply:
@JonDouglas & @nkolev92 - so which one are you suggesting to remove (the existing comment or the blog post comment)? The API stability is more about breaking changes, I'd assume. @zivkan would know more.

Reply:
We don't want to remove the API stability comment, it's about the blog one.